Packs/Legacy/Integrations/TruSTAR/TruSTAR.yml

```diff
@@ -618,7 +617,108 @@ script:
+      defaultValue: UNRESOLVED
+    outputs:
+    - contextPath: TruSTAR.Phishing.submissionId
+      description: The submission ID
+      type: string
+    - contextPath: TruSTAR.Phishing.title
+      description: Submission title
+      type: string
+    - contextPath: TruSTAR.Phishing.normalizedTriageScore
+      description: Submission triage score
+      type: number
+    - contextPath: TruSTAR.Phishing.context.indicatorType
+      description: Indicator type
+      type: string
+    - contextPath: TruSTAR.Phishing.context.sourceKey
+      description: Indicator source
+      type: string
+    - contextPath: TruSTAR.Phishing.context.normalizedSourceScore
+      description: Indicator score
+      type: number
+    description: Fetches all phishing submissions that fit the given criteria
+  - name: trustar-set-triage-status
+    arguments:
+    - name: submission_id
+      required: true
+      description: ID of the email submission
+    - name: status
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - CONFIRMED
+      - IGNORED
+      description: Submission status
+    description: Marks a phishing email submission with one of the phishing namespace
+      tags
+  - name: trustar-get-phishing-indicators
+    arguments:
+    - name: normalized_source_score
+      description: Intel score of the indicator
+      isArray: true
+      defaultValue: 0,1,2,3
+    - name: normalized_triage_score
+      description: Score of email submission
+      isArray: true
+      defaultValue: 0,1,2,3
+    - name: from_time
+      description: Start of time window (defaults to 7 days ago) (YYYY-MM-DD HH:MM:SS)
+    - name: to_time
+      description: End of time window (defaults to current time) (YYYY-MM-DD HH:MM:SS)
+    - name: status
+      description: A list of triage statuses for submissions; only email submissions
+        marked with at least one of these statuses will be returned. Options are 'UNRESOLVED',
+        'CONFIRMED', 'IGNORED'
+      isArray: true
+      defaultValue: UNRESOLVED,CONFIRMED,IGNORED
+    outputs:
+    - contextPath: File.Name
+      description: File name
+      type: string
+    - contextPath: File.MD5
+      description: File MD5
+      type: string
+    - contextPath: File.SHA1
+      description: File SHA1
+      type: string
+    - contextPath: File.SHA256
+      description: File SHA256
+      type: string
+    - contextPath: URL.Address
+      description: URL address
+      type: string
+    - contextPath: IP.Address
+      description: IP address
+      type: string
+    - contextPath: Account.Email.Address
+      description: Email address
+      type: string
+    - contextPath: RegistryKey.Path
+      description: Registry key path
+      type: string
+    - contextPath: CVE.ID
+      description: CVE ID
+      type: string
+    description: Get phishing indicators that match the given criteria.
+  dockerimage: demisto/trustar:1.0.0.7265
+  runonce: false
+  subtype: python3
```

Comment:
why did you change the python version? this might break things for customers on our env